ImageProcessing/README.md

```diff
@@ -0,0 +1,747 @@
+# Optional Module: Coordinate a serverless image processing workflow with AWS Step Functions
```

Comment:
I notice images in the changeset that aren't actually referenced in the README such as screenshots of the bucket creation processes. Probably makes sense to clean these up.